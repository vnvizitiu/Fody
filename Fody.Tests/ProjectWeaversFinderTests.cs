using System.IO;
using ApprovalTests;
using Moq;
using NUnit.Framework;

[TestFixture]
public class ProjectWeaversFinderTests
{
    [Test]
    public void NotFound()
    {
        var loggerMock = new Mock<BuildLogger>();
        loggerMock.Setup(x => x.LogDebug(It.IsAny<string>()));
        var logger = loggerMock.Object;
        var testDirectory = TestContext.CurrentContext.TestDirectory;
        var searchDirectory = Path.Combine(testDirectory, "FodyWeavers.xml");

        var weavingException = Assert.Throws<WeavingException>(() => ConfigFileFinder.FindWeaverConfigs(testDirectory, testDirectory, logger));
        Approvals.Verify(weavingException.Message.Replace(searchDirectory, "SearchDirectory"));
    }
}