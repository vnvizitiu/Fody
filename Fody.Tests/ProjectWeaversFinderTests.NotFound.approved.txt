Could not find path to weavers file. Searched 'SearchDirectory', 'SearchDirectory'.